pages/kidDetails/[id].js

```diff
@@ -0,0 +1,23 @@
+import useSWR from "swr";
+import { useRouter } from "next/router";
+import KidProfile from "@/components/KidProfile/KidProfile";
+
+export default function KidDetails() {
+  const router = useRouter();
+  const { id } = router.query;
+  const { data, isLoading } = useSWR(`/api/kids/${id}`);
+
```

Comment:
Somehere here would be a good place to put the `handleGoBack` function from the profile component and pass it down.